File: ce/snippets/csharp/CRMV8/marketing/cs/copydynamiclisttostatic.cs
// <snippetcopydynamiclisttostatic>



 // Copy the dynamic list to a static list.
 CopyDynamicListToStaticRequest copyRequest = 
     new CopyDynamicListToStaticRequest()
     {
         ListId = _dynamicListId
     };
 CopyDynamicListToStaticResponse copyResponse =
     (CopyDynamicListToStaticResponse)_serviceProxy.Execute(copyRequest);
 _staticListId = copyResponse.StaticListId;

 Console.WriteLine("Copied dynamic list to a static list.");

// </snippetcopydynamiclisttostatic>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/copydynamiclisttostatic.vb
' <snippetcopydynamiclisttostatic>



 ' Copy the dynamic list to a static list.
 Dim copyRequest As New CopyDynamicListToStaticRequest() With
     {
         .ListId = _dynamicListId
     }
 Dim copyResponse As CopyDynamicListToStaticResponse =
     CType(_serviceProxy.Execute(copyRequest), 
         CopyDynamicListToStaticResponse)
 _staticListId = copyResponse.StaticListId

 Console.WriteLine("Copied dynamic list to a static list.")

' </snippetcopydynamiclisttostatic>